test/project_types/script/layers/infrastructure/extension_point_repository_test.rb

```diff
@@ -8,7 +8,7 @@
   describe ".get_extension_point" do
     describe "when the extension point is configured" do
       Script::Layers::Infrastructure::ExtensionPointRepository.new
-        .send(:extension_points)
+        .send(:extension_point_configs)
         .each do |extension_point_type, _config|
```

Comment:
It feels here like we're testing the actual configuration rather than the component. Should this test live separately so that we can test the repository in isolation?

Reply:
Yeah good call, this does feel a little weird.